src/components/PermissionsManagement/PermissionsConst.js

```diff
@@ -97,6 +97,18 @@ export const permissionLabels = [
         description:
           'Gives the user permission to change the status of any user on the user profile page or User Management Page. "User Profile" -> "Green round button"',
       },
+      {
+        label: 'Toggle Invisibility Permission Self and Others',
+        key: 'toggleInvisibility',
+        description:
+          'Gives the user permission to change the invisibility toggle for themselves and others',
+      },
+      {
+        label: 'Handle Blue Squares',
+        key: 'infringementAuthorizer',
+        description:
+          'Gives the user permission to Create/Edit/Delete any blue square and assign them to any user.',
+      },
```

Comment:
```suggestion
```
This permission was separated into add/edit/delete permissions in in PRs [943](https://github.com/OneCommunityGlobal/HGNRest/pull/943)+[2248](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2248)

Reply:
Changes made. Please review again.